cmd/csi-snapshotter/main.go

```diff
@@ -73,8 +74,8 @@ var (
 )
 
 var (
-	version                = "unknown"
-	leaderElectionLockName = "external-snapshotter-leader-election"
+	version = "unknown"
+	prefix  = "external-snapshotter-leader"
```

Comment:
@msau42 Is there any limitation on the length of the lockName?  This prefix is a little longer.  Other than that, I'm fine with it.

Reply:
The attacher and resizer all use "external-" prefix. Probably we should just be the same to be consistent.

In terms of length, the name is used as the ObjectMeta.Name, which is 253 characters

Reply:
Sure.